restapi/src/main/java/io/stargate/web/docsapi/dao/DocumentDB.java

```diff
@@ -58,8 +58,12 @@
   public static final String EMPTY_OBJECT_MARKER = "EMPTYOBJ-bccbeee1-6173-4120-8492-7d7bafaefb1f";
   public static final String EMPTY_ARRAY_MARKER = "EMPTYARRAY-9df4802a-c135-42d6-8be3-d23d9520a4e7";
 
+  public static final String LEAF_VALUE_NAME = "leaf";
```

Comment:
I already wrote about this in #1235, any chance we can move to the solutions proposed there wrt constants?